components/src/core/drawer/drawer-nav-item.tsx

```diff
@@ -89,7 +89,7 @@ export const DrawerNavItem: React.FC<DrawerNavItemProps> = (props) => {
                 styles={{
                     root: Object.assign(
                         {
-                            paddingLeft: 16 * (depth > 1 ? depth - 1 : 0),
+                            paddingLeft: 32 * (depth > 1 ? depth - 1 : 0),
```

Comment:
the logic for React looks like this:
![image](https://user-images.githubusercontent.com/29152776/85278708-49b36400-b453-11ea-9999-78cd389842de.png)
Should we match that, or is there a compelling reason to bump the spacing to 32 at every level?

Reply:
This is matching the react one. The react component is bumping everything up by spacing(4) per depth

Reply:
ah, good catch. I missed the `spacing` part...looks good. 🦀

Reply:
I cannot unplay the Crab Rave song in my head now bc that emoji